addon/adapters/twitch-channel-feed.js

```diff
@@ -0,0 +1,8 @@
+import TwitchAdapter from 'ember-data-twitch/adapters/twitch';
+
+export default TwitchAdapter.extend({
+
+  pathForType(/* modelName */) {
+    return 'feed';
```

Comment:
Although, I think I'm going about `channel-feed` a bit wrong. That's the official name of the API resource, but it's actually concerned with returning a "post" object -- which is what I think we should be modelling. I can make another update to change `twitch-channel-feed` to `twitch-post`, if that sounds right.